spec/index.html

```diff
@@ -8421,6 +8421,168 @@ <h4>SPARQL Query</h4>
             one query level in the tree.</p>
         </section>
       </section>
+      <section id="algebraicSyntax">
+        <h3>Algebraic Syntax</h3>
+        <p>To define the evaluation semantics of SPARQL query strings
+          these strings are first translated into a more abstract, algebraic syntax.
+          This section defines the expressions that can be formed in this algebraic syntax,
+          and the translation of SPARQL query strings into this algebraic syntax is then defined
+          in Section&nbsp;<a href="#sparqlQuery" class="sectionRef"></a>.</p>
+        <p id="defn_AlgebraicQueryExpression">An
+          <a href="#defn_AlgebraicQueryExpression">algebraic query expression</a>
+          is defined recursively as follows:</p>
+        <ul>
```

Comment:
Presentation comment:

This might be better as a table. All the `<var>A</var> is an algebraic query expressions` etc can be written once then the presentation is more compact.